docker/build.Dockerfile

```diff
@@ -17,26 +17,20 @@ RUN wget -nv -O /tmp/geth.tar.gz ${GETH_URL_LINUX} && \
     rm geth.tar.gz
 
 
-ADD requirements*.txt /tmp/
-ADD constraints.txt /tmp/
+ADD requirements/requirements.txt /tmp/
```

Comment:
No, the requirements file is written to `/tmp`. 
This is just an optimization to prevent unnecessary re-installing of the deps (via docker layer caching) since they change much less frequently as the rest of the code.